src/main/java/city/parking/services/ParkingCostCalculationUtil.java

```diff
@@ -0,0 +1,56 @@
+package city.parking.services;
+
+import city.parking.entities.Money;
+import city.parking.entities.ParkingProcess;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.Currency;
+
+public class ParkingCostCalculationUtil {
+    private static final double REGULAR_COST_MULTIPLIER = 1.5;
+    private static final double COST_FOR_DISABLED_MULTIPLIER = 1.2;
+
+    private static final Currency PRIMARY_CURRENCY = Currency.getInstance("PLN");
+
+    private static final int ROUNDING_SCALE = 2; //number of decimal places of money amount to keep
+    private static final RoundingMode ROUNDING_MODE = RoundingMode.HALF_EVEN;
+
+    public static Currency getPrimaryCurrency(){
+        return PRIMARY_CURRENCY;
+    }
+
+    public static Money calculatePrimaryParkingCost(ParkingProcess process) {
+        int parkingTimeInHours = getParkingTimeInHours(process);
+        double primaryCost = process.isForDisabled() ? getDisabledCost(parkingTimeInHours) : getRegularCost(parkingTimeInHours);
```

Comment:
czy liczenie $$ w `double` to dobry pomysł?

Reply:
Powinno być na przykład BigDecimal jak w klasie Money